### PR TITLE
Member nft tab

### PR DESCRIPTION
**Pull request by eldiegod**
fixes: #1752

### CODE REVIEW

File: packages/atlas/src/views/viewer/MemberView/MemberView.styles.ts [hunk 1]
```diff
@@ -31,23 +31,38 @@ export const StyledTabs = styled(Tabs)`
 
 export const TabsContainer = styled.div`
   display: grid;
-  margin-bottom: ${sizes(8)};
+  padding-top: ${sizes(8)};
   gap: ${sizes(2)};
-  grid-template: 'tabs tabs tabs' 1fr 'search search search' auto 'sort sort sort' auto / 1fr 1fr;
+  grid-template: 'tabs tabs tabs' 1fr 'search search search' auto 'sort sort sort' auto / 1fr 1fr 'filter filter filter' auto / 1fr 1fr;
```
Comment (drillprop) [inline packages/atlas/src/views/viewer/MemberView/MemberView.styles.ts]: Something is very wrong on smaller screens: 
image

Second thing - can we just do this without declaring grid-areas? `  grid-template: 1fr auto auto / 1fr 1fr;`? should work the same in this case, right?

File: packages/atlas/src/views/viewer/MemberView/MemberView.styles.ts [hunk 2]
```diff
@@ -2,7 +2,7 @@ import styled from '@emotion/styled'
 
 import { MembershipInfo } from '@/components/MembershipInfo'
 import { Tabs } from '@/components/Tabs'
-import { media, oldColors, sizes } from '@/styles'
+import { cVar, media, oldColors, sizes, transitions } from '@/styles'
```
Comment (drillprop) [inline packages/atlas/src/views/viewer/MemberView/MemberView.styles.ts]: let's get rid of `oldColors` in this file.

**Review by drillprop — CHANGES_REQUESTED** [review]

File: packages/atlas/src/views/viewer/MemberView/MemberView.styles.ts (hunk 1 above)
Comment (eldiegod) [reply packages/atlas/src/views/viewer/MemberView/MemberView.styles.ts]: Grid areas is just easier due the mobile grid layout difference where tabs take 2 columns, I think they are fine it was just not properly indented/wrong

File: packages/atlas/src/views/viewer/MemberView/MemberView.styles.ts [hunk 3]
```diff
@@ -13,41 +13,46 @@ export const NotFoundMemberContainer = styled.div`
...
+    'tabs tabs' auto
+    'sort filter' auto / 1fr auto;
+  background-color: #000;
 
   ${media.sm} {
     align-items: center;
-    border-bottom: solid 1px ${oldColors.gray[700]};
-    gap: ${sizes(8)};
-    grid-template: 1fr / 1fr 160px;
+    box-shadow: ${cVar('effectDividersBottom')};
```
Comment (drillprop) [inline packages/atlas/src/views/viewer/MemberView/MemberView.styles.ts:36]: Just one thing left.
image

**Review by drillprop — CHANGES_REQUESTED** [review]

**Review by drillprop — APPROVED** [review]

**Review by drillprop — APPROVED** [review]

**Pull request merged**